src/pages/userManagement/UserManagerCollaborationEditTable.jsx

```diff
@@ -82,23 +83,32 @@ export default function UserManagersCollaborationEditTable({
         userTwoGuid: get(member2, 'guid'),
         viewStatusOne: get(member1, 'viewState'),
         viewStatusTwo: get(member2, 'viewState'),
-        // editStatusOne: get(
-        //   member1,
-        //   'editState',
-        // ),
-        // editStatusTwo: get(
-        //   member2,
-        //   'editState',
-        // ),
       };
+      // editStatusOne: get(
+      //   member1,
+      //   'editState',
+      // ),
+      // editStatusTwo: get(
+      //   member2,
+      //   'editState',
+      // ),
     });
   }
-  const tableFriendlyData = tranformDataForCollabTable(data);
+  const tableFriendlyData = tranformDataForCollabTable(
+    inputData,
+  )?.filter(
+    collab =>
+      get(collab, 'viewStatusOne') !== revokedPermission ||
+      get(collab, 'viewStatusTwo') !== revokedPermission,
+  );
```

Comment:
maybe this extra logic should go inside `tranformDataForCollabTable`?

Reply:
Done. Good point.